kotlin-css/src/commonMain/kotlin/kotlinx/css/CSSBuilder.kt

```diff
@@ -169,7 +169,7 @@ class CSSBuilder(
     }
 
     fun prefix(selector: String, block: RuleSet) {
-        "$selector &"(block)
+        "$selector &&"(block)
```

Comment:
```suggestion
        // Temporarily using && here because of a bug introduced in version 5.2: https://github.com/styled-components/styled-components/issues/3244#issuecomment-687676703
        "$selector &&"(block)
```